requirements/edx/base.txt

```diff
@@ -90,7 +90,7 @@ python-dateutil==2.1
 # We can also remove the fix to auth_url in third_party_auth/saml.py when that fix is included upstream.
 python-social-auth==0.2.12
 
-pytz==2015.2
+pytz>=2016.7
```

Comment:
This won't necessarily gaurantee an upgrade.  In most cases we don't build a new virtualenv, we re-use the existing virtual environment.  In this case, if `pytz` is already installed it will not automatically upgrade even when you need it to.